src/components/DateInput/ClearButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import AbstractButton, { AbstractButtonProps } from '../AbstractButton';
+
+const TextButton: React.FC<AbstractButtonProps> = props => {
+  return (
+    <AbstractButton
+      textDecoration="underline"
+      fontSize="small"
+      color="gray-300"
+      sx={{
+        '&:hover': {
+          textDecoration: 'none',
+        },
```

Comment:
optional: I don't think we have a hover behavior in the designs, but I'm not sure as well

Reply:
That's the "TextButton" from Panther
https://github.com/panther-labs/panther-enterprise/blob/master/web/src/components/buttons/TextButton/TextButton.tsx